test/parallel/test-http-information-processing.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
-require('../common');
+const common = require('../common');
 const assert = require('assert');
 const http = require('http');
+const Countdown = require('../common/countdown');
 
 const test_res_body = 'other stuff!\n';
-let processing_count = 0;
+const countdown = new Countdown(3, common.mustCall(() => server.close()));
```

Comment:
The `Countdown` is already a `mustCall` the wrapping is therefore not necessary :-)

Reply:
Done